### ISSUE TITLE
Document dependencies

### ISSUE THREAD

**Issue by andrewjli** [federicot/ng-message-center#3]
Hello again! I think it would be a good idea to document the libraries that this module relies upon in the README, namely jQuery and Bootstrap JS. I realise that if you work with a Bootstrap project, they're probably always included, but as I was using the Bootstrap-UI library, I thought I could drop the various extra JS libraries I wasn't using. Turns out I couldn't.

**Comment by federicot** [comment]
Right, actually I would like to remove any dependencies and let the user decide how to style and "animate" messages.

https://github.com/federicot/ng-message-center/tree/unit-tests

I've added unit tests and removed both dependencies. Still I have to clean the bootstrap html.
I also still have to allow the user to define their own template and message close callback function.
